### DISCUSSION TITLE
libvips: Difference between v8.14 and v8.13 when autorotating some PNGs

### DISCUSSION THREAD

**Discussion by dloebl** [libvips/libvips#3269] [General]
As libvips v8.14 added support for EXIF metadata to PNG load/save, there is a minor inconsistency when autorotating some PNGs with an EXIF orientation tag compared to v8.13:

[test.png](https://user-images.githubusercontent.com/34104349/211286599-bb6fd027-b379-4f1a-ac24-7dfe4c0157d0.png)
`vips autorot test.png out.jpg`
**Note:** The orientation tag seems to be erroneously set on the affected PNGs.

| v8.13 | v8.14 |
| - | - |
| ![813](https://user-images.githubusercontent.com/34104349/211286524-7075e2d5-efd5-4070-971f-48c5b6ca9f87.jpg) | ![814](https://user-images.githubusercontent.com/34104349/211286544-d573f356-0b15-4daa-bb84-2c0955a08594.jpg) |

Not sure if that's something that requires changes on libvips side, but I just wanted to mention this as most image viewers/browsers ignore the EXIF orientation tag on PNGs. For now we resolved this difference by just removing the EXIF orientation tag on `pngload`.

**Comment by jcupitt** [comment]
Huh that's true, there is a change in behaviour. But I think it's inevitable if we add EXIF support to PNG, and we clearly should, so I think it's probably OK (as you say!).
